src/VirtualClient/VirtualClient.Actions/SysbenchOLTP/SysbenchOLTPExecutor.cs

```diff
@@ -267,6 +280,19 @@ public bool SysbenchInitialized
                 }
             }
 
+            public bool ExecutablesInitialized
```

Comment:
Let's not expose this in parameters. Saving in state is ok, but user shouldn't need to know it

Reply:
gotcha -- does that mean making this private?

Reply:
no I mean making it a property in the MySQL state, so user don't need to know whether they need to pass in this parameter